bioblend/galaxy/invocations/__init__.py

```diff
@@ -10,9 +10,23 @@ def __init__(self, galaxy_instance):
         self.module = 'invocations'
         super().__init__(galaxy_instance)
 
-    def get_invocations(self):
+    def get_invocations(self, history_id=None, user_id=None, include_terminal=True, limit=None, view='collection', step_details=False):
```

Comment:
```suggestion
    def get_invocations(self, workflow_id=None, history_id=None, user_id=None, include_terminal=True, limit=None, view='collection', step_details=False):
```

Also need to document and use `workflow_id` below.

Reply:
workflow_id wasn't specified in the original function, is it a url parameter?

Reply:
No, it's the same as `history_id` and `user_id` in this case.